R/get_delay_estimate.R

```diff
@@ -13,7 +13,7 @@
 #' @param max_delay Integer indicating the maximum delay to estimate, in units
 #'   of the delay. The default is to use the whole reporting triangle,
 #'   `ncol(triangle) -1`.
-#' @param n_history Integer indicating the number of reference dates to be
+#' @param n_history_delay Integer indicating the number of reference dates to be
```

Comment:
Can you remind me what the purpose of this name change is? It feels unrelated?

Reply:
is it because of the argument naming choices you make higher up? Generally I think it is better to do naming etc at the lowest level as a self contained thing (i.e they shouldn't know what they are going to be used for) and then higher up rename and pass as needed for the context of that function. This avoids all you code depending on each other in complicated ways that differ from just function usage

Reply:
Yeah this was to distinguish between `n_history_delay` and `n_history_uncertainty` but per the discussion above I think these should be variables in the code and this should just be `n`?